index.html

```diff
@@ -447,14 +447,89 @@ <h3>Event Model</h3>
                         </ul>
                     </section>
                 </section>
+                <section>
+                  <section>
+                    <h3>Adaptive experiments</h3>
+                  </section>
+                  <section>
+                    <p>Feedback Paths</p>
+                    <ul>
+                      <li> data quality checks </li>
+                      <li> prompt / real-time analysis to steer experiment
+                        <ul>
+                          <li> "human-in-the-loop" </li>
```

Comment:
I think human vs computer applies to both data quality checks and experiment steering. Maybe make a separate bulleted list rather than sub-bullets on this point.